Documentation/docker_guide.md

```diff
@@ -0,0 +1,47 @@
+# Running etcd under Docker
+
+The following guide will show you how to run etcd under Docker using the [static bootstrap process](op-guide/clustering.md#static).
+
+In order to expose the etcd API to clients outside of the Docker host you'll need use the host IP address of the etcd container. Please see [`docker inspect`](https://docs.docker.com/engine/reference/commandline/inspect) for more detail. Or specify `--net=host` flag to `docker run` command to skip placing the container inside of a separate network stack.
```

Comment:
s//you'll need/,/
s/detail/detail on how to get the IP address/
s/Or/Alternatively,/